.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Summary
+[PLEASE REMOVE] Describe the goal of this PR and any add any relevant context.
+
+[PLEASE REMOVE] If this PR closes an issue, please add a `Fixes #<issue-id>` line.
+
+[PLEASE REMOVE] If this PR introduces a fix or feature that should be the upcoming release notes, please add the following "Release notes: <area>" label.
```

Comment:
If we're going to start using these labels, we need to document them so authors know how to pick one. https://github.com/pytorch/executorch/blob/main/CONTRIBUTING.md#pull-requests would be a good place for it, and we should point to that from here.

And if you're proposing that we do this, we'll need to educate everyone on the Edge team to start using them, and to ensure that they're set before approving PRs from external contributors. It's worth writing up a workplace post pointing to this PR (which can also include the CONTRIBUTING.md updates) and asking for feedback.